ivy/stateful/module.py

```diff
@@ -593,7 +593,9 @@ def build(
                 ivy.Container.cont_assert_identical_structure(
                     [created_n_found, v_from_constructor]
                 )
-                self.v = v_from_constructor
+                # propagate v_from_constructor to child variables
+                created_n_found.cont_set_at_key_chains(v_from_constructor, inplace=True)
```

Comment:
I think we can instead use `cont_inplace_update` here instead because we know that the structure of `v_from_constructor` and `created_n_found` are the same till we reach this point, and hence we don't need the handling used to update specified keychains from the `target_dict` in the `created_n_found` container (as all key_chains need to be updated).

Reply:
Just went through its implementation, actually, `cont_set_at_key_chains` or `cont_overwrite_at_keychains` would be the fastest and most appropriate for this case, given its implementation. Since we already confirm the structure of both of these containers to be the same, these functions are designed to quickly update the values, these functions were designed for this specific reason.